json-struct.c

```diff
@@ -1663,9 +1663,7 @@ static void gen_struct(FILE *fp, struct jc_struct *s)
   }
   fputs(" */\n", fp);
 
-  if (t_alias)
-    fprintf(fp, "typedef ");
-  fprintf(fp, "struct %s {\n", t);
+  fprintf(fp, "typedef struct %s {\n", t);
```

Comment:
do this mean every `struct` has an alias?

Reply:
Yes, this change adds a default typedef if a alias isn't specified.